creator-cards/ricardo-perez.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ricardo Josue Perez Altamirano",
+  "type": "CREATOR",
+  "tags": ".NET, C#, ASP.NET, Blazor, ASP .NET Core, Azure, .NET MAUI, Xamarin",
+  "github": "https://github.com/RicardoJosuePerezAltamirano",
+  "channels": [
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/c/RicardoJosue"
+    },{
+      "name": "Sessionize",
+      "url": "https://sessionize.com/ricardo-josue-perez-altamirano/"
+    },{
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ricardojosue/"
+    },{
+      "name": "Twitter",
+      "url": "https://twitter.com/RicardoJosue04"
+    },{
+      "name": "Blog",
+      "url": "https://dev.to/ricardojosue04"
+    }
+  ],
+  "feeds": [{
+    "type": "dotnet",
+    "url": "https://www.youtube.com/feeds/videos.xml?channel_id=UC5Rh7fFBjc8_EBEeEXg67aw"
+  }],
+  "projects": [
+    {
+      "name": "DemoServiciosNativosMAUI",
+      "description": "Demo del consumo de servicios nativos en android usando .NET MAUI",
```

Comment:
Can you please give us an english description?

Reply:
ready in commit 37fa261a5b8ee1d10e89d6dad442afa3cf3342e5